src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__description {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__product-code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--secondary-color);
+  margin-bottom: 16px;
+}
+
+.product-card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.product-card__reviews-count {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price-name {
+  font-size: 12px;
+  line-height: 16px;
+  color: var(--secondary-color);
+}
+
+.product-card__price-tag {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__buy-link {
+  align-self: center;
+  transition: 0.3s;
+  font-weight: 700;
+  font-size: 14px;
+  width: 166px;
+  height: 40px;
+  text-decoration: none;
```

Comment:
You have a reset-file for css. So such rules as  `text-decoration: none ` is a reseting style. Move it there